### PR TITLE
Updated read me

### PR DESCRIPTION
**Pull request by codewithkushagra**
Added yarn and node docs like

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,9 @@ https://github.com/techhub-community/techhub-website/tree/dev) branch.
 
 - [Learn NextJS ](https://nextjs.org/docs/getting-started) - Next.js docs.
```
[the remark this answers is not in the document]
Comment (codewithkushagra) [reply README.md:61]: Yeah, sure 
I did found some good video resource will just it

[the remark this answers is not in the document]
Comment (dntandan) [reply README.md:61]: And do add some get started with node and yarn blog from a very good blog in docs under Learn Yarn and node section ( Seaprate)

[the remark this answers is not in the document]
Comment (codewithkushagra) [reply README.md:61]: Okay will do that too

[the remark this answers is not in the document]
Comment (codewithkushagra) [reply README.md:61]: @dntandan  Should make a section for Learn More(blog) instead and put the blog for node and yarn in that section

[the remark this answers is not in the document]
Comment (dntandan) [reply README.md:61]: yeah go ahead.

[the remark this answers is not in the document]
Comment (codewithkushagra) [reply README.md:80]: @dntandan I have made the changes and have committed them should I make a new pull request
or the changes I committed will be reflected in this request only?

File: README.md [hunk 2]
```diff
@@ -36,14 +36,33 @@ https://github.com/techhub-community/techhub-website/tree/dev) branch.
 5. Your `PR` will be reviewed and will be merged to the develop branch only.
 6. Upon ample PR merge to develop branch, all PRs will be merged to `main` branch & will be live.
 
+## Learn More(Blogs)
```
Comment (dntandan) [inline README.md:39]: LGTM

**Review by dntandan — APPROVED** [review]

**Comment by dntandan** [comment]
Merging it now.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dntandan) [reply README.md:80]: If you have created a PR and make some changes and commit them again it gets added to that PR automatically. No need to worry.

**Pull request merged**